contracts/external/interfaces/IPermit2.sol

```diff
@@ -18,6 +18,13 @@ interface IPermit2 {
         uint256 requestedAmount;
     }
 
+    function permitTransferFrom(
```

Comment:
Do we not want to use the permitWitnessTransferFrom function? I'm not sure myself, just asking the question. Maybe @dohaki would know better

Reply:
You're right. We do want it https://github.com/across-protocol/contracts/pull/782/commits/4f3b2d6b094548efeeb11dacab40acd11f67bccd